### ISSUE TITLE
`linkerd.io/reference` could redirect to latest reference docs

### ISSUE THREAD

**Issue by cratelyn** [linkerd/website#1849]
as of writing, the latest reference documentation lives at https://linkerd.io/2.16/reference/. other versions' documentation can be accessed at `:{version}/reference/`.

it would be nice if https://linkerd.io/reference/ redirected clients to the latest reference, much like how https://linkerd.io/docs redirects to

**Comment by travisbeckham** [comment]
We currently have a `/2/` redirect in place that forwards any doc to the latest version.

For example, https://linkerd.io/2/reference/ redirects to the latest version of the reference docs.

Is this sufficient, or is there a reason we need to have a specific redirect for the reference section at the root?

**Comment by cratelyn** [comment]
i wouldn't have known to put a `/2/` in that path before `/reference`. i'm not strongly attached to this though, so i'll go ahead and close this as "not planned" for now.

**Comment by travisbeckham** [comment]
> i wouldn't have known to put a `/2/` in that path

Yeah, no one would know to do this. We primarily just use it in the docs when we want a link to always point to the latest version.

I wish the real location of the latest docs was always at the root with an alias for `/{latest-version}/*`. Then only older versions would be located under a `/{version}/` directory. I'll see if there's a way to make this work without breaking existing links.

**Comment by travisbeckham** [comment]
Now that #2116 is merged, we have a permanent url for the [latest reference page](https://linkerd.io/docs/reference/).
